apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/hooks/useTransactionMonitoringAlertsLogic/useTransactionMonitoringAlertsLogic.tsx

```diff
@@ -20,6 +20,7 @@ export const useTransactionMonitoringAlertsLogic = () => {
     search: searchValue,
     sortDir,
     sortBy,
+    entityType: 'transaction',
```

Comment:
`entityType` is expected to be `business` or `individual`. Is `transaction` an entity type in our system elsewhere?

Reply:
I dont have context on this.
@alonp99